pypeit/waveimage.py

```diff
@@ -134,9 +147,43 @@ def __repr__(self):
         txt = '<{:s}: >'.format(self.__class__.__name__)
         return txt
 
```

Comment:
Need a way to pass an optional argument here for pseudo masters. See my comment before.

Reply:
Addressed.